crates/core/src/db/queries/twitter_handle_name_service.rs

```diff
@@ -32,3 +33,31 @@ where
         .optional()
         .context("Failed to load twitter handle")
 }
+
+const GET_MULTIPLE_HANDLES_QUERY: &str = r"
+SELECT A.*
+FROM twitter_handle_name_services A
+INNER JOIN (
+    SELECT wallet_address, MAX(write_version) as write_version
+    FROM twitter_handle_name_services
+    WHERE wallet_address = ANY($1)
+    GROUP BY wallet_address
+) B on A.wallet_address = B.wallet_address AND A.write_version = B.write_version
```

Comment:
Why are we querying write version?

Reply:
Misguided attempt to avoid returning multiple entries for a single wallet.